spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaStreamsDefaultConfiguration.java

```diff
@@ -31,10 +33,14 @@
  * {@link KafkaStreamsDefaultConfiguration#DEFAULT_STREAMS_CONFIG_BEAN_NAME} is present
  * in the application context. Otherwise a {@link UnsatisfiedDependencyException} is thrown.
  *
+ * Also collects {@link KafkaStreamsCustomizer}s to be applied just after {@link KafkaStreams}
+ * created and before {@link KafkaStreams#start()}.
```

Comment:
Sounds like something is missed in this class.
However we discussed with you about some conventional `KafkaStreamsCustomizer`. Similar to what we have now for the `StreamsConfig` with the `DEFAULT_STREAMS_CONFIG_BEAN_NAME`.

Then again: this JavaDoc is not related anymore and has to be changed respectively